<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="11418" started-at="2018-06-14T10:58:39Z" finished-at="2018-06-14T10:58:51Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11418" started-at="2018-06-14T10:58:39Z" finished-at="2018-06-14T10:58:51Z">
      <class name="com.js.JSErrorsDemo">
        <test-method status="PASS" signature="init()[pri:0, instance:com.js.JSErrorsDemo@23ceabc1]" name="init" is-config="true" duration-ms="3847" started-at="2018-06-14T16:28:39Z" finished-at="2018-06-14T16:28:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="yifySite()[pri:0, instance:com.js.JSErrorsDemo@23ceabc1]" name="yifySite" duration-ms="7167" started-at="2018-06-14T16:28:43Z" finished-at="2018-06-14T16:28:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- yifySite -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.js.JSErrorsDemo@23ceabc1]" name="tearDown" is-config="true" duration-ms="385" started-at="2018-06-14T16:28:50Z" finished-at="2018-06-14T16:28:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.js.JSErrorsDemo -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
